sdk/stdlib-sys/src/stdlib/crypto/hashes.rs

```diff
@@ -199,7 +199,7 @@ pub fn sha256_hash_2to1(input: [u8; 64]) -> [u8; 32] {
 /// This maps to the `std::crypto::rpo::hash_memory` procedure in the Miden stdlib.
 ///
 /// # Arguments
-/// * `elements` - A slice of field elements to be hashed
+/// * `elements` - A Vec of field elements to be hashed
 #[inline]
 pub fn hash_elements(elements: Vec<Felt>) -> Digest {
```

Comment:
Question: Is there any reason not to have this function take a slice rather than a `Vec`? Seems odd to pass a Vec by value here.

Reply:
Nice! Done - https://github.com/0xMiden/compiler/issues/690